source/numerics.tex

```diff
@@ -8228,7 +8228,7 @@
 refers.
 \end{itemdescr}
 
-\rSec3[gslice.array.comp.assign]{\tcode{gslice_array}}
+\rSec3[gslice.array.comp.assign]{\tcode{gslice_array} computed assignment}
```

Comment:
This should presumably be "compound assignment, not "computed assignment", per [expr.ass].